### DISCUSSION TITLE
Support importing additional file types (HAR, Swagger, WSDL)

### DISCUSSION THREAD

**Discussion by BradNut** [usebruno/bruno#881] [Ideas]
Hello! It would be helpful if at some point Bruno could support importing files of type HAR, Swagger, and WSDL. Given I'd much rather use Bruno but Postman and Insomnia support these file types.

I frequently work with these types and it would be a great benefit if Bruno had it at some point.
They didn't seem to be on the roadmap from what I could see so I just wanted to start a conversation and see what people thought.

Thanks!

**Comment by JvE-iO** [comment]
+1 for WSDL

**Reply by rusak47** [reply]
@helloanoop @rreyn-bruno  I'm really enjoying Bruno! I was wondering if WSDL import support is on the horizon. It would be a fantastic addition for API testing. I didn't see it on the current roadmap but would love to know if it's planned.

**Comment by rreyn-bruno** [comment]
This is a really logical request. Not sure if you meant Swagger=OpenAPI, but we can import OAS files.  HAR and WSDL make sense as well. We're planning to focus on importers (this and others) in the start of 2025. We hope to release a public roadmap this month, and I'll see if this is prominent on it.

**Reply by BradNut** [reply]
Sorry yes OpenAPI is really what I meant given Swagger uses OpenAPI.
